src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/commands.py

```diff
@@ -14,6 +14,18 @@ def __call__(self, poller):
         return result.properties.authorization_policies
 
 
+class LinkedHubUpdateResultTransform(LongRunningOperation):  # pylint: disable=too-few-public-methods
+    def __call__(self, poller):
+        result = super(LinkedHubUpdateResultTransform, self).__call__(poller)
+        return result.properties.iot_hubs
```

Comment:
This isn't the correct use of a transform. If your commands now return a poller, the CLI will automatically do the polling for you and return the object.  Transform is used if you needed to change the structure of the returned object.  Transform callables should accept a parameter called `result` (typically) that is the object being returned.